Ecosystem Partners/getting-started-with-minecraft-pe-server-blueprint.md

```diff
@@ -0,0 +1,144 @@
+
+4. Configure your server information within the client:
+  * Enter a Server Name in the first field.
+  * Enter your public IP address in your second field.
+  * Click the Add Server button once your information has been typed in.
+  ![Configure Server Image](../images/ecosystem-minecraft-pe-server-4.png)
+
+5. Click on your server to start playing
+  ![Click Server Image](../images/ecosystem-minecraft-pe-server-5.png)
+
+6. The game should connect to your server and you can begin playing immediately.
+  ![You Connected Image](../images/ecosystem-minecraft-pe-server-6.png)
+
+  * If you cannot connect, make sure you entered the Public IP address correctly and ensure the 2x firewall ports are open on the Public IP.
+
+#### Access the server console for administrative purposes
+After your Blueprint deploys successfully, please follow these instructions to access your server:
+
+  1. Check your email to obtain the servers private IP Address information, or obtain it from the Control Portal
+  2. Establish a VPN connection to your data center and then SSH to your Minecraft Server's Private IP. Login as the user minecraftpe with password you set when you deployed the Blueprint.  If you experience any issues logging in as the minecraftpe user, please see the FAQ below.
+  3. Connect to your Minecraft  console by running this command as the minecraftpe user: screen -r
+  4. To disconnect from the java console: press "CTRL A + Z" (A then Z).
+  5. To stop (and save) the server, login to the console and type save-all then stop.  For more information, please consult the Minecraft PE documentation.
+
+### Pricing
+The costs associated with this Blueprint deployment are for the CenturyLink Cloud infrastructure only.  There are no Mojang or Microsoft license costs or additional fees bundled in.
+
+### Frequently Asked Questions
+
+#### What does the Blueprint do?
+1. Provision a VM with 1CPU and 4GB of RAM running Ubuntu 14 Linux x64
+2. Run the 'Install Minecraft Server on Linux x64' Script Package
+
+#### What does the 'Install PE Minecraft Server on Linux x64' Script Package do?
+1. Add a user called minecraftpe to the Linux OS.  This user will run the Minecraft PE server software so the inecraft PE Server will not run as root.  The minecraftpe users password will be set to the password provided during deployment. 
+2. Download the latest Minecraft PE software from the Mojang website to ~minecraftpe (/home/minecraftpe)
+3. Append information from the user to the Minecraft PE server.properties configuration file
+4. Starts the Minecraft PE software as the minecraftpe user. 
+
+#### What Firewall ports are required to be opened?
+* 2x firewall ports need to be open in order to play that game over the internet: Port 19132 using TCP and Port 19132 using UDP
+
+#### How do I start the Minecraft PE server application?
+*. Login via ssh as the minecraftpe user and run the command:  screen ./start.sh
```

Comment:
Also in general for things that you type in fi you wrap them with tick (`) characters it helps them stand out. 

example: type `./start.sh` into the command prompt